pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -172,11 +172,25 @@ type Config struct {
 	Networking   NetworkingOpts
 }
 
+func logcfg(cfg Config) {
+	glog.V(3).Infof("AuthURL: %s", cfg.Global.AuthURL)
```

Comment:
@jichenjc can we move it to `V(5)`? since `V(3)` will log always?

Reply:
ok, will do ,thanks for the reminder